Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,424 @@
+
+
+    @logger
+    def add_exclusion(exclusion_type, exclusion_data, exclusion_description, groups_id):
+        groups_ids_list = [group_id.strip() for group_id in groups_id.split(',')]
+        json_params = {
+            'type': exclusion_type,
+            'description': exclusion_description,
+            'data': exclusion_data,
+            'appliedGroupsIds': groups_ids_list
+        }
+        response = session.post(BASE_URL + '/api/exclusions',
+                                json=json_params)
+        if response.status_code is not 200:
+            return_error('Error while adding exclusion. More information: {}, {}'.format(response.status_code, response.reason))
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': response.json(),
+            'HumanReadable': tableToMarkdown('Exclusion \'{}\' was added'.format(exclusion_description), response.json(), headerTransform=pascalToSpace, removeNull=True)
+        })
+
+
+    @logger
+    def add_vaccine(vaccine_name, vaccine_desc, monitor_only):
+        json_params = {'name': vaccine_name,
+                       'description': vaccine_desc,
+                       'isMonitorOnly': monitor_only,
+                       'type': 'mutex'}
+        response = session.post(BASE_URL + '/api/vaccination',
+                                json=json_params)
+        if response.status_code is not 200:
+            return_error('Error while adding a vaccine. More information: {}, {}'.format(response.status_code, response.reason))
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': response.json(),
+            'HumanReadable': tableToMarkdown('Vaccination \'{}\' was added'.format(vaccine_desc), response.json(), headerTransform=pascalToSpace, removeNull=True)
+        })
+
+
+    @logger
+    def search(search_url, search_param, search_condition, search_value):
+        json_params = {'filters': [{'param': search_param,
+                                    'condition': search_condition,
+                                    'value': search_value}]}
+        response = session.post(search_url,
+                                json=json_params)
+        if response.status_code is not 200:
+            return_error('Error while perfroming search for {}. More information: {}, {}'.format(search_url.rsplit('/')[1], response.status_code, response.reason))
+
+        results = {
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': response.json(),
+            'HumanReadable': tableToMarkdown('Search results for \'{}\''.format(search_url.split('/')[-1]), response.json(), headerTransform=pascalToSpace, removeNull=True)
+        }
+
+        if response.json():
+            results['EntryContext'] = create_entry_context(search_url, response.json()[0])
```

Comment:
Why only the first element?

Reply:
Also need to add `Endpoint` or `Process` to the context object, e.g., `"Minerva.Process": create_entry_context`

Reply:
I treated the first element only, as the output retrieved from our Management Console server appears to be a list (with a single dictionary in it)